### PR TITLE
Feature/add transaction

### PR DESCRIPTION
**Pull request by p-enz**
Add new Form for adding a new Transaction.
Implement new function in useStore
Add a new Route
Solve a bug that was implement by mistake

### CODE REVIEW

File: src/db_backup.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			},
+			{
+				id: nanoid(),
+				date: '08-06-2022',
+				change: 50.0,
+				note: 'visit grandma',
+			},
+		],
+	},
+];
```
Comment (JessicaLoers) [inline src/db_backup.js:64]: Safety first, but please delete the backup file.😁

File: src/db.js [hunk 2]
```diff
@@ -12,12 +12,14 @@ const db = [
 				date: '07-06-2022',
 				change: -40.31,
```
Comment (JessicaLoers) [inline src/db.js:13]: Maybe it makes sense to have an `expense: negative number ` , `income: positive number`  🤔
That way your entries stay clean and more specific.

File: src/components/AddTransaction/index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+					onChange={event => {
+						setTransaction({...transaction, note: event.target.value});
+					}}
+					type="text"
+					aria-label="Enter a decribtive note for that transaction"
+					placeholder="e.g. online shopping"
+				></input>
+
+				<label htmlFor="categorieMenu">Select a fitting category</label>
+				<input
```
Comment (JessicaLoers) [inline src/components/AddTransaction/index.js:65]: Why don't you use `<select>` for the selection of categories?
[MDN <select>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select)

File: src/components/AddTransaction/index.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+import {useState} from 'react';
+import {useParams} from 'react-router-dom';
+
+import useStore from '../../hooks/useStore';
+import {Form} from '../AddNewAccount/styled';
+
+export default function AddTranscation() {
+	const [transaction, setTransaction] = useState({
+		date: '',
+		change: '',
```
Comment (JessicaLoers) [inline src/components/AddTransaction/index.js:10]: Maybe it makes sense to have an `expense: negative number ` , `income: positive number`  🤔
That way your entries stay clean and more specific.

**Review by JessicaLoers — COMMENTED** [review]
**Well done, DaPh** - this is your team name from now on 😁
Leave you some comments.

File: src/db.js (hunk 2 above)
Comment (p-enz) [reply src/db.js:13]: Thanks for the advice. We where not sure how we want to implement this. We thought maybe through buttons or so.  
We will have a look 😃

File: src/components/AddTransaction/index.js (hunk 3 above)
Comment (p-enz) [reply src/components/AddTransaction/index.js:65]: We weren´t aware of that :D

File: src/db_backup.js (hunk 1 above)
Comment (p-enz) [reply src/db_backup.js]: The file is deleted :)

**Pull request merged**